src/shared/lib/google-ad-sense.tsx

```diff
@@ -10,20 +11,32 @@ declare global {
 }
 
 interface GoogleAdSenseProps {
-  slot?: string
   className?: string
 }
-const GoogleAdSense = ({ slot, className }: GoogleAdSenseProps) => {
+const GoogleAdSense = ({ className, ...props }: GoogleAdSenseProps) => {
   const adRef = useRef<HTMLModElement>(null)
 
+  // 데스크탑: 수동 광고 사용
+  // 태블릿/모바일: 자동 광고 사용 (Google이 <ins> 태그 없이 광고 자동 삽입)
+  const { isDesktop } = useDeviceUtils()
+  const [isDesktopView, setIsDesktopView] = useState(false)
```

Comment:
```suggestion
  const isDesktopView = isDesktop
```
[P1] `isDesktop`이 항상 최신 상태를 반환한다면 `isDesktopView`는 useState를 사용하지 않아도 될 것 같아요!

Reply:
피드백 감사해요!
그런데 이렇게 수정하려니 pc환경에서 Hydration mismatch 에러가 발생해서 현 상태 유지해야할 것 같은데 혹시 다른 방법 있을까요?😢

> Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used
>   <ins
>   ref={{current:null}}
>   className="adsbygoogle block h-[37.5rem] w-40 place-content-center"
>   data-ad-client="ca-pub-4289003874294919"
>   data-ad-slot="9725653724"
>   data-ad-format="auto"
>   data-full-width-responsive="true"
>   >
SSR 시에는 isDesktop이 false라 null이 렌더되지만, 클라이언트에서는 true로 판단돼 hydration 시점에 DOM이 달라져서 에러가 발생한 것 같아요!